skins/oasis/modules/BodyController.class.php

```diff
@@ -80,7 +80,12 @@ public static function isGridLayoutEnabled() {
 	 * @return Boolean
 	 */
 	public static function isOasisBreakpoints() {
-		global $wgOasisBreakpoints, $wgRequest;
+		global $wgOasisBreakpoints, $wgRequest, $wgLanguageCode, $wgOasisBreakpointsDE;
+
+		//For now we want to disable breakpoints for German wikis if not turn on explicitly.
+		if ( strtolower( $wgLanguageCode ) == 'de' && empty( $wgOasisBreakpointsDE ) ) {
```

Comment:
I assume we'll remove this condition block after 71Media adjust their styles. Therefore, my suggestion is to put here `//TODO: remove when 71Media adjust their styles` too.